examples/3chambers/3chambers.py

```diff
@@ -204,9 +192,9 @@ def step(n, loop_n, plot=True):
     print(graph.node_nb(), graph.edge_nb())
 
     population = nngt.NeuralPop(with_models=False)
-    population.create_group(range(100), "chamber_1", )
-    population.create_group(range(100, 200), "chamber_2")
-    population.create_group(range(200, 300), "chamber_3")
+    population.create_group("chamber_1", range(100))
```

Comment:
These changes are inconsistent with ``2chambers/2chambers.py``, if you're using up-to-date NNGT versions, they should not be correct: function signature is

    nngt.NeuralPop.create_group(self, neurons, name, ...)